local/bin/py/build/configurations/pull_config.yaml

```diff
@@ -413,7 +413,7 @@
   - repo_name: browser-sdk
     contents:
     - action: pull-and-push-file
-      branch: master
+      branch: main
```

Comment:
this was already fixed in live, had to update it in my branch before the change went out though.